docs/database/domain-database/groups.md

```diff
@@ -15,7 +15,7 @@ Group management is implemented using so-called linking tables. Hence, it is pos
 
 - associate domains (and clients!) with any number of groups,
 - manage lists together with groups,
-- use the same groups for black- and whitelisted domains at the same time.
+- use the same groups for deny and allowlist domains at the same time.
```

Comment:
```suggestion
- use the same groups for deny- and allowlist domains at the same time.
```

Reply:
I agree this is ugly and it looks like a typo without any following characters, but I think this is called "suspended hyphen", used when you list similar compound words. 

I'm not sure about its usage rules. 
I don't know if it should be used here and I'm not sure if it is wrong to omit it... after all, I'm not a native English speaker.

If you think something should be changed, I have a suggestion (use the complete words):

```suggestion
- use the same groups for denylist and allowlist domains at the same time.
```

Reply:
We use the suspended hyphen in German a lot  - I thought it might be the same in English. But when it looks strange to a native speaker just ignore it :-)

Reply:
I don't think it's wrong and, actually, it is wrong *without* it. Because the former is "deny" and the latter "allowlist". Without the extra `-` it is not clear the former is meant to be a short form of "allowlist". But I'm likewise not going to argue (much) against  the native speaker here.

Reply:
I'm no expert in Grandma, of course, just something looked off to me.

I think maybe to be ultra clear we can just be verbose and go `denylist and allowlist` as RD suggested

Reply:
Balls to it. I've just read the wiki article about suspended hyphens. Lets do that then, I'll keep my nose out!

Reply:
And now I've read it again - that would be used if there was already a hyphen in the word i.e `deny-list` / `allow-list`, but there is not.

Back to my (RD's) original plan